src/actions/targetActions.js

```diff
@@ -32,6 +52,7 @@ export const addTarget = target =>
   async (dispatch) => {
     try {
       const targetResponse = await TargetApi.createTarget(target);
+      dispatch(deleteTargetSuccess({}, -1));
```

Comment:
Why are you passing an empty object and -1?
Also, I saw that you are repeating `deleteTargetSuccess({}, -1)` in other files, so maybe it's better declare deleteTargetSuccess function as `deteleTargetSuccess(index = -1, target = {})` and call it in other line as:
`deleteTargetSuccess()`
`deleteTargetSuccess(index)`
`deleleTargetSuccess(index, targetResponse) `